Client/Frontend/Browser/Tab Management/Tab.swift

```diff
@@ -583,23 +583,34 @@ class Tab: NSObject {
         webView?.stopLoading()
     }
 
-    func reload() {
-        // If the current page is an error page, and the reload button is tapped, load the original URL
-        if let url = webView?.url, let internalUrl = InternalURL(url), let page = internalUrl.originalURLFromErrorPage {
-            webView?.replaceLocation(with: page)
-            return
-        }
-
-        if let _ = webView?.reloadFromOrigin() {
-            print("reloaded zombified tab from origin")
-            return
-        }
-
-        if let webView = self.webView {
-            print("restoring webView from scratch")
-            restore(webView)
-        }
-    }
+	func reload(bypassCache: Bool = false) {
+		// If the current page is an error page, and the reload button is tapped, load the original URL
+		if let url = webView?.url, let internalUrl = InternalURL(url), let page = internalUrl.originalURLFromErrorPage {
+			webView?.replaceLocation(with: page)
+			return
+		}
+		
+		if bypassCache {
+			let reloadRequest = URLRequest(url: url!,
+				cachePolicy:NSURLRequest.CachePolicy.reloadIgnoringLocalAndRemoteCacheData,
+				timeoutInterval: 10.0)
+
+			if let _ = self.webView?.load(reloadRequest) {
+				print("Reloaded the tab from originating source, ignoring local cache.")
```

Comment:
If we want debug information we don't use print but `browserLog.debug("Your log information")`, please replace the three prints with that.

Reply:
Apologies, was following the other examples where print was used. I replaced all 3 print statements with `browserLog.debug()`. Please refer to latest commit